src/test/scala/com/ibm/couchdb/spec/Fixtures.scala

```diff
@@ -56,30 +87,10 @@ trait Fixtures {
     name = "test-design",
 
     views = Map(
-      FixViews.names → CouchView(map =
-        """
-        |function(doc) {
-        | emit(doc.doc.name, doc.doc.name);
-        |}
-        """.stripMargin),
-      FixViews.reduced → CouchView(map =
-        """
-        |function(doc) {
-        | emit(doc._id, doc.doc.age);
-        |}
-        """.stripMargin, reduce =
-        """
-        |function(key, values, rereduce) {
-        | return sum(values);
-        |}
-        """.stripMargin),
-      FixViews.compound → CouchView(map =
-        """
-          |function(doc) {
-          | var d = doc.doc;
-          | emit([d.age, d.name], d);
-          |}
-        """.stripMargin)),
+                 FixViews.names → FixViews.namesView,
```

Comment:
Just a minor thing, could you please replace → with -> for consistency with other places? :)